src/bot.py

```diff
@@ -75,40 +123,43 @@ def tweet_loop(self, api):
                 else:
                     api.update_status(self.generate_batch())
                     logging.info("Tweeted out a new flower bed!")
-                    logging.info("The next tweet is scheduled to be made in {} minutes".format(cooldown))
+                    logging.info("The next tweet is scheduled to be made in {} seconds".format(cooldown))
                     time.sleep(cooldown)
-            except:
-                print(sys.exc_info())
-                logging.critical("Tweeting failed due to ratelimit. Waiting {} more minutes.".format(cooldown))
-                time.sleep(cooldown)
+                    raise tweepy.TweepError("error")

```

Comment:
Why are you raising an error? The syntax isn't right here, you'd want to type it like this;
```
time.sleep(cooldown)
raise tweepy.TweepError("error")
```

But this just raises an error for literally no reason.

Reply:
Apologies. I was experimenting with exception, nothing to do with actual code.